src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -37,14 +83,13 @@ const DomainFragment = ({
   withoutRecent = false
 }) => {
   const navigation = useNavigation();
-  const [myId, setMyId] = React.useState('');
   const [discovery, discoveryDispatch] = React.useContext(Context).discovery;
+  const [activeSections, setActiveSections] = React.useState([]);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of local state (`activeSections`) and context (`discovery`) is a good practice for managing state in React. However, make sure to handle cases where `activeSections` might be null or undefined throughout your code.
```javascript
const [discovery, discoveryDispatch] = React.useContext(Context).discovery;
const [activeSections, setActiveSections] = React.useState([]);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->